src/test/java/steps/SignUpSteps.java

```diff
@@ -1,63 +1,24 @@
 package steps;
-
-import config.UserConfig;
-import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
-import io.cucumber.java.en.When;
-import pages.BasePage;
-import pages.SignUpPage;
-
-public class SignUpSteps {
-    private BasePage basePage;
-    private SignUpPage signUpPage;
-
-    @Then("click button {string}")
-    public void clickButtonTryTestRailForFree(String arg0) {
-        basePage.clickButton();
-    }
-
-    @And("input Web Address")
-    public void inputWebAddress() {
-        signUpPage.webAddress.sendKeys(UserConfig.WEB_ADRESS);
-    }
-    @And("input First Name")
-    public void inputFirstName() {
-        signUpPage.webAddress.sendKeys(UserConfig.FIRST_NAME);
-    }
-    @And("input Last Name")
-    public void inputLastName() {
-        signUpPage.webAddress.sendKeys(UserConfig.LAST_NAME);
-    }
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import pages.HomePage;
 
-    @And("in the field Your Country choose a value {string}")
-    public void inTheFieldYourCountryChooseAValue(String arg0) {
-    }
-
-    @And("input Work Email")
-    public void inputWorkEmail() {
-        signUpPage.webAddress.sendKeys(UserConfig.EMAIL);
-    }
-
-    @And("input Organization")
-    public void inputOrganization() {
-        signUpPage.webAddress.sendKeys(UserConfig.ORGANIZATION);
-    }
+import static com.codeborne.selenide.Selenide.*;
 
-    @And("in the field Expected Users choose a value {string}")
-    public void inTheFieldExpectedUsersChooseAValue(String arg0) {
-    }
+public class SignUpSteps extends BaseSteps{
+    private static final String HOME_PAGE = "https://www.gurock.com/testrail/";
+    private static final String TRY_TEST_RAIL_FOR_FREE = "//*[@id='et_builder_outer_content']//*[@class='btn btn-medium btn-green'][@role = 'button']";
 
-    @Then("choose input with text {string}")
-    public void chooseInputWithText(String arg0) {
+    @Given("At Home page")
+    public void openHomePageForLogin(){
+       open(HOME_PAGE);
+       // homePage.openHomePage(HOME_PAGE);
     }
 
-    @When("User clicks button {string}")
-    public void userClicksButtonCreateTestRailTrial() {
-        basePage.buttonSignUp();
+    @Then("click button {string} on HomePage")
+    public void clickButtonForLogin() {
```

Comment:
public void clickButtonForLogin(String button)

Reply:
@Then("click button 'Try TestRail for Free' on HomePage")
    public void clickButtonForLogin() {
        homePage.clickButtonForFree();
    }